src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx

```diff
@@ -95,13 +95,19 @@ const DotPlotPage = (props) => {
   } = useSelector((state) => state.genes.properties.views[plotUuid] || {});
 
   const cellSets = useSelector(getCellSets());
+
+
+
   const {
     loading: cellSetsLoading,
     error: cellSetsError,
     hierarchy: cellSetHierarcy,
     properties: cellSetProperties,
   } = cellSets;
 
+
+  const clusterNames = Object.values(cellSetProperties).map((el) => el.name);
```

Comment:
I don't think you should get the values here `clusterNames` here. Implementing it here would add another parameter to be passed to `fetchPlotDataWork`. I think it would be better to attach `clusterNames` inside the body request body that's generated for this plot.